haxelib/aze/display/TilesheetEx.hx

```diff
@@ -51,21 +53,26 @@ class TilesheetEx extends Tilesheet
 		#if flash
 		bmps = new Array<BitmapData>();
 		#end
+		this.rotation = rotation;
+		if (rotation != 0)
+			rotations = new Array<Bool>();
 	}
 
 	#if flash
-	public function addDefinition(name:String, size:Rectangle, bmp:BitmapData)
+	public function addDefinition(name:String, size:Rectangle, bmp:BitmapData, rotated:Bool = false)

```

Comment:
You should pass the exact Float rotation instead of a Bool - it would allow any rotation and reduce `getRotation` logic to simply return the final value.